hack/make-rules/test-cmd-util.sh

```diff
@@ -1050,48 +1050,6 @@ run_kubectl_create_filter_tests() {
   set +o errexit
 }
 
-run_kubectl_apply_deployments_tests() {
```

Comment:
@liggitt I will remove parts of the test so that it only verifies ability to explicitly send a null field is still working.
Yes, the defaulting behavior must not change in the existing API groups.